packages/draft-js-import-markdown/src/__tests__/stateFromMarkdown-test.js

```diff
@@ -61,6 +61,47 @@ describe('stateFromMarkdown', () => {
       },
     ]);
   });
+  it('should correctly handle images with alt text', () => {
+    const src = 'https://google.com/logo.png';
+    const alt = 'The Google Logo';
+    let markdown = `![${alt}](${src})`;
+    let contentState = stateFromMarkdown(markdown);
+    let rawContentState = convertToRaw(contentState);
+    let blocks = removeKeys(rawContentState.blocks);
+    expect({
+      ...rawContentState,
+      blocks,
+    }).toEqual({
+      entityMap: {
+        // This is necessary due to flow not supporting non-string literal property keys
```

Comment:
Oh, I think the trick here that I used in the past is:
```js
{ [0]: "foo" }
```
because IIRC Flow is fine when the number literal is in a computed property.

Reply:
I just copied that from the old tests :wink:

Reply:
Oops, my bad. You're right!